app/views/casa_cases/_thank_you_modal.html.erb

```diff
@@ -2,9 +2,18 @@
   <div class="modal fade" id="thank_you" data-backdrop="static" data-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
     <div class="modal-dialog modal-dialog-centered">
       <div class="modal-content">
-        <div class="modal-body">
+        <div class="modal-header">
           <%= "Case contact was successfully created. #{thank_you_message}" %>
         </div>
+        <% if when_do_we_have_court_dates(@casa_case) == 'future' %>
```

Comment:
**nitpick:** you are using `"future"` in the module and `'future'` here. I tend to think being consistent with quotes is better. That said, that's obviously me being annoying, so up to you listening to this or not.